ERCS/erc-7710.md

```diff
@@ -8,7 +8,7 @@ status: Draft
 type: Standards Track
 category: ERC
 created: 2024-05-20
-requires: 1271
+requires: 1271, 7579
```

Comment:
I thought we were avoiding saying that this requires 7579 because while it uses one interface from 7579 it doesn’t actually require The integrading contract account to support the entirety of that interface, causing confusion that results in extra work.

Reply:
oh right! Reverted